src/gennylib/include/gennylib/PoolFactory.hpp

```diff
@@ -45,7 +45,9 @@ class PoolFactory {
     };
 
 public:
-    PoolFactory(std::string_view uri);
+    using ApmCallback = std::function<void(const mongocxx::events::command_started_event&)>;
```

Comment:
add note that this is temporary. Let's create a TIG ticket to track the CDRIVER ticket and add that ticket number to the comment